cmds/core/wget/wget_test.go

```diff
@@ -40,55 +41,56 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 var tests = []struct {
+	name    string
 	flags   []string // in, %[1]d is the server's port, %[2] is an unopen port
 	url     string   // in
 	content string   // out
 	retCode int      // out
 }{
 	{
-		// basic
+		name:    "basic",
 		flags:   []string{},
 		url:     "http://localhost:%[1]d/200",
 		content: content,
 		retCode: 0,
 	}, {
-		// ipv4
+		name:    "ipv4",
 		flags:   []string{},
 		url:     "http://127.0.0.1:%[1]d/200",
 		content: content,
 		retCode: 0,
-	}, /*{ TODO: travis does not support ipv6
-		// ipv6
+	}, /*{ CircleCI does not support ipv6
```

Comment:
could you just use a // comment here for lines 62-67? I find this as written a bit hard to parse..